src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier;
+    private final Random random;
+    private final int ballMaxNum = 100;
```

Comment:
```suggestion
    private static final int BALL_MAX_NUMBER = 100;
```
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names